src/screens/LoginPage/LoginPage.module.css

```diff
@@ -2,7 +2,18 @@ html,
 body {
   overflow-x: hidden !important;
 }
+
 /* ===== Login Modal CSS Start ===== */
+
+/* = Nasiru's hide and show password style = */
```

Comment:
Could you remove this comment? It is unnecessary and doesn't do anything. Also, this is a public repository and it is not advised to add your name to the comment. The git commit with your name is enough. Make sure there is a consistent line break.